commands/motor/MotorSet.java

```diff
@@ -1,41 +1,37 @@
 package org.usfirst.frc4904.standard.commands.motor;
 
-import java.util.Set;
 
 import org.usfirst.frc4904.standard.LogKitten;
 import org.usfirst.frc4904.standard.subsystems.motor.Motor;
 import edu.wpi.first.wpilibj.SpeedController;
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.Subsystem;
-import java.util.Set;
-import java.util.HashSet;
+import edu.wpi.first.wpilibj.command.Command;
```

Comment:
This needs to be converted to wpilibj2.

Reply:
Not a file we should have edited so no need to fix.